src/os/bluestore/BlueStore.cc

```diff
@@ -8921,6 +8921,7 @@ int BlueStore::queue_transactions(
   } else {
     osr = new OpSequencer(cct, this);
     osr->parent = posr;
+    osr->shard_hint = posr->shard_hint;
```

Comment:
we can probably call shard_hint.hash_to_shard and preserve its result in osr instance instead. This saves us a bit...